server/game/cards/07-WotW/SpeakToTheHeart.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class SpeakToTheHeart extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'give +1 political to a character for each faceup province',
+            condition: () => this.game.isDuringConflict(),
+            max: AbilityDsl.limit.perConflict(1),
+            target: {
+                cardCondition: card => card.isFaction('unicorn'),
+                gameAction: ability.actions.cardLastingEffect(context => ({
+                    effect: ability.effects.modifyPoliticalSkill(context.player.getNumberOfOpponentsFaceupProvinces())
+                }))
+            },
+            effect: 'give {0} +1{1} for each faceup non-stronghold province their opponent controls',
+            effectArgs: () => ['political']
```

Comment:
Not a big deal, but may be helpful to say how much has been added.  Maybe something like 'player1 uses Speak to the Heart to give Border Rider +1[pol] for each faceup non-stronghold province their opponent controls (+3[pol])' ?

This is more of my personal preference though so people may disagree.

Reply:
What ! I shamelessly copied this from your Iuchi Daiyu ! :p

Reply:
Haha.  Whoops!  Guess I should update that :P